### DISCUSSION TITLE
How to get Series information for a release

### DISCUSSION THREAD

**Discussion by VinylVault** [joalla/discogs_client#150] [General] [answered]
Some releases belong to Series, is there a way you can get the series information?

**Comment by prcutler** [comment]
Can you share a release ID that contains a series?  I think `.fetch` might work.  This release returns an empty list, probably because it's not part of a series:

```py
>>> import discogs_client
>>> d = discogs_client.Client('TestREPL/0.1')
>>> release = d.release(1026691)
>>> print(release.id)
1026691
>>> print(release.fetch('series'))
[]
>>>

**Reply by VinylVault** [reply]
https://www.discogs.com/release/31125881-Various-Now-Yearbook-93 is one example, I was hoping that it would be natively available tbh

**Reply by prcutler** [reply]
It looks like the data is there using `fetch`:

```py
>>> import discogs_client
>>> d = discogs_client.Client('TestREPL/0.1')
>>> release = d.release(31125881)
>>> print(release.id)
31125881
>>> print(release.fetch('series'))
[{'name': 'Now Yearbook', 'catno': "'93", 'entity_type': '2', 'entity_type_name': 'Series', 'id': 2583262, 'resource_url': 'https://api.discogs.com/labels/2583262'}]
>>>
```

**Reply by VinylVault** [reply]
ok, so because im trying to add this in a script, how would i do something if that fetch succeeds?

would this work?

```
variablename = release.fetch("series")
if variablename:
   # do something
```

**Reply by JOJ0** [reply]
should work. if it's not a series the fetch will return an empty list, which is falsy.

**Reply by VinylVault** [reply]
@JOJ0 could Series please be added to the general release info

**Reply by JOJ0** [reply]
yes, let's please open a feature request issue to track it. thanks

**Reply by VinylVault** [reply]
> yes, let's please open a feature request issue to track it. thanks

#151 but i cant add a tag for some reason

**Comment by prcutler** [comment, marked as answer]
Without knowing what you want to script, a basic Python program might look like this.  You could wrap it in a try / except if a given release doesn't have a series, but you just want to make sure you account for the list and the dictionary inside it:

```py
import discogs_client

d = discogs_client.Client('SeriesTest/0.1')

release = d.release(31125881)
series_info = release.fetch('series')
print(series_info)

series_id = series_info[0]['id']
series_name = series_info[0]['name']

print('Series name: ', series_name, 'Series ID: ', series_id)
```

And that returns:
```py
[{'name': 'Now Yearbook', 'catno': "'93", 'entity_type': '2', 'entity_type_name': 'Series', 'id': 2583262, 'resource_url': 'https://api.discogs.com/labels/2583262'}]
Series name:  Now Yearbook Series ID:  2583262
```
